classes/author.rb

```diff
@@ -0,0 +1,15 @@
+class Author
+  attr_accessor :name, :items
```

Comment:
First name and Last names should be separately stored in two individual variables